.gradient/workflows/training-pipeline.yaml

```
@@ -0,0 +1,225 @@
+            type: dataset
+            with:
+              ref: prepared-training-data
+            
+      - name: model-training
+        uses: script@v1
+        needs: [data-preparation]
+        with:
+          script: |
+            # Model training step
+            echo "Starting model training..."
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)